providers/amazon/src/airflow/providers/amazon/aws/waiters/bedrock.json

```diff
@@ -68,6 +68,140 @@
                     "state": "failure"
                 }
             ]
+        },
+        "batch_inference_complete": {
+            "delay": 120,
+            "maxAttempts": 75,
+            "operation": "GetModelInvocationJob",
+            "acceptors": [
+                {
+                    "matcher": "path",
+                    "argument": "status",
+                    "expected": "Completed",
+                    "state": "success"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "status",
+                    "expected": "Failed",
+                    "state": "failure"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "status",
+                    "expected": "Stopped",
+                    "state": "failure"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "status",
+                    "expected": "PartiallyCompleted",
+                    "state": "failure"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "status",
+                    "expected": "Expired",
+                    "state": "failure"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "status",
+                    "expected": "Submitted",
+                    "state": "retry"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "status",
+                    "expected": "InProgress",
+                    "state": "retry"
+                },
+                {
+                    "matcher": "path",
+                    "argument": "status",
+                    "expected": "Stopping",
+                    "state": "retry"
```

Comment:
If `stopped` is failure, should we consider this one as failure as well? To fail fast. This could be an assumption which can lead to some other issues so feel free to disagree :)

Reply:
I'll think on it a bit, but yeah, I think you are right.   That shouldn't have any side effects and would cut wait time down.  Nice catch.